src/pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fi.vrk</groupId>
     <artifactId>rest-adapter-service</artifactId>
-    <version>0.0.13-SNAPSHOT</version>
+    <version>0.0.13</version>
```

Comment:
Should the version point directly to the next snapshot-version, since the release version has already been released?